frontend/src/metabase/query_builder/components/QueryDownloadWidget/QueryDownloadWidget.tsx

```diff
@@ -40,7 +40,7 @@ const QueryDownloadWidget = ({
   });
 
   return (
-    <Popover opened={isPopoverOpen} onClose={() => setIsPopoverOpen(false)}>
+    <Popover opened={isPopoverOpen} onChange={() => setIsPopoverOpen(false)}>
```

Comment:
Although this seems to be working, the code does not seem right: on any popover state change we always set isPopoverOpen to false. Wdyt if we pass `onChange={setIsPopoverOpen}`?

Reply:
Right, that would make sense. There is also `onDismiss`... It's really unclear to me the relationship between these props on Controlled Popovers.